contentcuration/contentcuration/frontend/channelEdit/components/QuickEditModal/EditBooleanMapModal.vue

```diff
@@ -171,6 +181,8 @@
       cancelAction: 'Cancel',
       updateDescendantCheckbox:
         'Apply to all resources, folders, and subfolders contained within the selected folders.',
+      addAdditionalCategoriesDescription:
```

Comment:
This string exists in`shared/translators.js` already as `addAdditionalCatgoriesDescription`, so let's use that one!